Plans/dispatch-notify-on-complete.md

```
@@ -0,0 +1,364 @@
+	"sessionId": "<tabId>",
+	"tabId": "<tabId>",
+	"callbackId": "cb_01J...",
+	"notifyOnComplete": "<caller-agent-id>"
+}
+```
+
+### Callback prompt template
+
+Default wrapper, with `--callback-prompt` overriding the whole body:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)